salt/orch/update.sls

```diff
@@ -321,11 +326,12 @@ etcd-setup:
     - require:
       - salt: {{ worker_id }}-apply-haproxy
 
-# Perform any migratrions after services are started
+# Perform any migrations after services are started
 {{ worker_id }}-update-post-start-services:
   salt.state:
     - tgt: '{{ worker_id }}'
     - sls:
+      - kubelet.uncordon
```

Comment:
All the states here are called by their _stage_ (`update-post-start-services`) instead of _what they do_ (to `uncordon`). I would try to follow the convention...

Reply:
Right, fixed.